package.json

```diff
@@ -46,7 +46,8 @@
     "vue-the-mask": "^0.11.1",
     "vuetify": "^2.3.12",
     "vuex": "^3.5.1",
-    "vuex-router-sync": "^5.0.0"
+    "vuex-router-sync": "^5.0.0",
+    "yarn": "^1.22.10"
```

Comment:
Was it added by accident @datamel ? Yarn shouldn't be a runtime dependency for Cylc UI I think

Reply:
yes, I have muddled up a rebase...just fixing now :)